indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/models/Nym.kt

```diff
@@ -0,0 +1,54 @@
+package com.luxoft.blockchainlab.hyperledger.indy.models
+
+import com.luxoft.blockchainlab.hyperledger.indy.utils.SerializationUtils
+
+data class NymResponse(
+        val op: String,
+        val result: Result
+) {
+    data class Result(
+            val `data`: String?,
+            val dest: String,
+            val identifier: String,
+            val reqId: Long,
+            val seqNo: Int?,
+            val state_proof: StateProof?,
+            val txnTime: Int?,
+            val type: String
+    ) {
+        fun getData(): Data? = `data`?.run { Data.fromString(this) }
+    }
+
+    data class Data(
+            val dest: String,
+            val identifier: String,
+            val role: String,
+            val seqNo: Int,
+            val txnTime: Int,
+            val verkey: String
+    ) {
+        companion object {
+            fun fromString(str: String): Data = SerializationUtils.jSONToAny(str)
+        }
+    }
+
+    data class StateProof(
+            val multi_signature: MultiSignature,
```

Comment:
Our object mapper is configured to translate snake_case JSON into camelCase fields and vice versa. 
If for some reason it doesn't work, there is a `@JsonProperty` annotation for such stuff.

Reply:
It was auto-generated from JSON file, why do we need to make logic more complex?